File: El Mamoun/Lv 1/Problem 15.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

/*
  Proble 15: Write a program to calculate rectangle area and print it on the screen.
*/

namespace Problem_15
{
    internal class Program
    {
        static void ReadNumbers(ref float Length,ref float Width)
        {
            
            Console.WriteLine("Pleae Enter a Length?: ");
            Length = Convert.ToSingle(Console.ReadLine());

            Console.WriteLine("Pleae Enter a Width?: ");
            Width = Convert.ToSingle(Console.ReadLine());

        }
        static float CalRectangeArea(float Length,float Width)
        {
            return (Length * Width);
        }
        static void PrintArea(float Area)
        {
            Console.WriteLine("The Area of Rectange is: " + Area);
        }


        static void Main(string[] args)
        {

            float Length = 0, Width = 0;
            ReadNumbers(ref Length,ref Width);

            PrintArea(CalRectangeArea(Length,Width));

            Console.ReadKey();
        }
    }
}

File: El Mamoun/Lv 1/Problem 15.cpp
#include<iostream>
using namespace std;

/*
  Proble 15: Write a program to calculate rectangle area and print it on the screen.
*/

void ReadNumbers(float &Length, float &Width)
{
	cout << "\nPlease Enter a Length? ";
	cin >> Length;

	cout << "\nPlease Enter a Width? ";
	cin >> Width;
}

float CalCulateRectangeArea(float A, float B)
{
	return float(A * B);
}

void PrintResult(float Area)
{
	cout << "\n Area Of Rectange is = " << Area << endl;
}

int main()
{

	float Length, Width;

	ReadNumbers(Length, Width);

	PrintResult(CalCulateRectangeArea(Length, Width));

	return 0;
}